packages/teleport/src/Kubes/EventDialog/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import EventDialog from './EventDialog';
```

Comment:
the file name `EventDialog` tells me it is a dialog for audit events (if you go to our audit logs screen, you'll see a button that will open a dialog to see the details of the event), but this dialog is for connecting to a kube cluster